NBISweden/urdr
# fix: activity name labels

## Related issue(s) and PR(s)
This PR closes https://github.com/NBISweden/urdr/issues/861



## Type of change

 
## List of changes made

- Made the activity show up in the "favourite issue" rows

## Screenshot of the fix

<img width="952" alt="Screenshot 2023-10-27 at 14 42 03" src="https://github.com/NBISweden/urdr/assets/3052170/07d3ae6a-cf1d-4e8f-ac18-222c7b2d58ae">

## Testing

- Restart urdr container
- Check your favourite issues and make sure the label is displayed

## Definition of Done checklist
